kubevirt/step1.md

```diff
@@ -0,0 +1,49 @@
+### Wait for the cluster to be ready
+
+Before we can start, we need to wait for the Kubernetes cluster to be ready
+
+#### Deploy KubeVirt
+
+Deploy KubeVirt operator using latest Kubevirt Version
```

Comment:
typo fixed.

about the contents we're reusing the materials we had for the minikube part (https://kubevirt.io/quickstart_minikube/) 

Would it be good to have a 'FAQ' like in metal3 site? so that those terms are explained at our website directly and users can check them without having to 'detail' in each other document?

Reply:
It's probably worth improving the quickstart in the website, and a FAQ might be a nice addition too, but let's handle that in the website repo.

But regardless of that, I still think it's 6 words is a bit too short of an explanation of what we're doing here.

Reply:
Operator was expanded to:

[^1]: An Operator is a method for packaging, deploying and managing Kubernetes applications, read more at [CoreOS Operators](https://coreos.com/operators/)


Is this not enough?

Reply:
My suggestion for [^1] is:

>  [^1] An Operator is a method of packaging, deploying and managing a Kubernetes application. A Kubernetes application is an application that is both deployed on Kubernetes and managed using the Kubernetes APIs and kubectl tooling. To be able to make the most of Kubernetes, you need a set of cohesive APIs to extend in order to service and manage your applications that run on Kubernetes. You can think of Operators as the runtime that manages this type of application on Kubernetes. If you want to learn more about Operators you can check the CoreOS Operators website: https://coreos.com/operators/